src/utils/cache.js

```diff
@@ -0,0 +1,17 @@
+import path from 'path'
+import { md5sum, findNodeModulesPath, getPackageName } from '.'
+
+export function createCacheKey(fn, opts) {
+  if (typeof opts !== 'string') {
+    opts = JSON.stringify(opts)
+  }
+
+  return md5sum(`${fn}-x-${opts}`)
+}
+
+export async function getCachePath(filePath = '') {
+  const modulesPath = await findNodeModulesPath()
+  const name = await getPackageName(modulesPath)
```

Comment:
@pi0 I implemented this getPackageName function but looking at it now I dont think its very useful actually. Probably easier if it was always `tib`?